README.md

```diff
@@ -56,6 +56,16 @@ To get started, please visit our [documentation](https://docs.paradedb.com).
 
 ParadeDB and its extensions are available as commercial software for installation on self-hosted Postgres deployment and via Docker and Kubernetes as standalone images. For more information, including enterprise features and support, please [contact us by email](mailto:sales@paradedb.com).
 
+### Using the Setup Script
+
+For an interactive setup run the following in your terminal.
+
+```bash
+curl -O https://raw.githubusercontent.com/paradedb/paradedb/dev/deploy.sh && bash deploy.sh
```

Comment:
Running this on my mac gives me:

```bash
curl -O https://raw.githubusercontent.com/paradedb/paradedb/dev/deploy.sh && bash deploy.sh

  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
100    14  100    14    0     0     90      0 --:--:-- --:--:-- --:--:--    91
deploy.sh: line 1: 404:: command not found
```

I then did `./deploy.sh` instead and I get:

```bash
zsh: permission denied: ./deploy.sh
```

Could you `chmod +x deploy.sh`? Maybe we should do `./`, since not every computer will have Bash